src/main/java/edu/byu/cs/autograder/PassoffTestGrader.java

```diff
@@ -107,19 +118,20 @@ protected TestAnalyzer.TestNode runTests() {
         // Process cannot handle relative paths or wildcards,
         // so we need to only use absolute paths and find
         // to get the files
-        String chessJarWithDeps = new File(stageRepoPath, "shared/target/shared-jar-with-dependencies.jar").getAbsolutePath();
+        String chessJarWithDeps = new File(stageRepoPath, module + "/target/" + module + "-jar-with-dependencies.jar").getAbsolutePath();
 
         ProcessBuilder processBuilder = new ProcessBuilder()
                 .directory(stageTestsPath)
-//              .inheritIO() // TODO: implement better logging
                 .command("java",
                         "-jar",
                         standaloneJunitJarPath,
-                        "--class-path", ".:" + chessJarWithDeps + ":" + junitJupiterApiJarPath,
+                        "--class-path",
+                        ".:" + chessJarWithDeps + ":" + junitJupiterApiJarPath + ":" + passoffDependenciesPath,
                         "--scan-class-path",
                         "--details=testfeed");
 
         try {
+            System.out.println("Compiled command: " + String.join(" ", processBuilder.command().toArray(new String[0])));
```

Comment:
For noise purpose I think this should be a `LOGGER.info`

Reply:
Another thing I meant to remove, oops. Thanks for noticing!